### DISCUSSION TITLE
Bare-metal install on raspberry Pi of  "jokob-sk/NetAlertX "

### DISCUSSION THREAD

**Discussion by mh5bl** [netalertx/NetAlertX#630] [Q&A]
I have a Raspberry Pi Model B Plus Rev 1.2 which I was running pi-hole I have install **leiweibau/Pi.Alert** but this Raspberry Pi hardware is not powerful enough for leiweibau/Pi.Alert and **pi.hole**
So my plan is to buy a new Raspberry Pi run **pi.hole** and **jokob-sk/NetAlertX** and uses the old Pi as a Secondary DNS running pi-hole.
So my question is what Bare-metal Raspberry Pi specification will run **jokob-sk/NetAlertX** ????
( I do not wish to use docker. )

**Comment by jokob-sk** [comment]
Hey there,

I can only say that I tested it on the Pi4 4gb (docker) and it works fine. I don't have unfortunately other models to test this on. 

Maybe someone else from the community can answer.
